.github/actions/spell-check/patterns.txt

```diff
@@ -0,0 +1,5 @@
+(?:0[Xx]|U\+|#)[a-f0-9A-FGgRr]{2,}[Uu]?[Ll]{0,2}\b
+addNSECRecordToLW.*DNSName.*powerdnt.com.*QType::NS.*res->d_records
+data:[a-zA-Z=;,/0-9+]+
+BOOST_CHECK_EQUAL\(b64, "[a-zA-Z=;,/0-9+]+"
```

Comment:
This covers things like:
https://github.com/PowerDNS/pdns/blob/c1fdf910b8fac26233157ee85d0d6e895c6ec9f8/pdns/test-signers.cc#L255